### PR TITLE
feature(database): General information on Databases

### LINKED ISSUE #219: T-SQL Documentation
**Document T-SQL**
We currently have very little documentation on T-SQL syntax differences and each SQL database. It is currently difficult make an informed decision on SQL databases based on a project scope.

Page in question: https://github.com/Shift3/standards-and-practices/blob/main/best-practices/database/README.md

**Describe the solution you'd like**
We should add some documentation to T-SQL as well as compare/contrast each SQL Database. 

- [ ] PostgreSQL
- [ ] MSSQL
- [ ] MySQL/MariaDB
- [ ] Oracle Database
- [ ] IBM DB2

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
1. Add a description on databases
2. Add link to comparison of databases

## Purpose
Users should be able to get more information on databases. 

## Approach
The linked resource also provides a general non-biased (in my opinion) description of the major databases. 

## Learning
I tried finding a comparison online that included many of the databses we use at Bitwise. Unfortunately, most of the resources seemed heavily biased and didn't include a nice pro/con setup. 

I didn't get a lot of feedback from the technical discussions channel, so I thought I would throw this out there and see what people think!

Closes #219

### CODE REVIEW

File: best-practices/database/README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# DATABASE
+# Databases
+A database is an organized collection of data, generally stored and accessed electronically from a computer system. Where databases are more complex they are often developed using formal design and modeling techniques. 
+
+At Shift3, we generally use T-SQL and NoSQL databases in our projects. More information can be found [here](https://www.altexsoft.com/blog/business/comparing-database-management-systems-mysql-postgresql-mssql-server-mongodb-elasticsearch-and-others/).
```
Comment (Karvel) [inline best-practices/database/README.md:4]: You might want to update the Shift3 reference.

File: best-practices/database/README.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+
+Feel free to contact architect@bitwiseindustries.com if you are deciding on which database to use.
```
Comment (Karvel) [inline best-practices/database/README.md:6]: I would word this more strongly. New projects are supposed to go through a pass with an architect.

**Review by Karvel — CHANGES_REQUESTED** [review]
Looks good. I think the Shift3 reference should be updated since it would need to be updated soon anyway.

File: best-practices/database/README.md (hunk 1 above)
Comment (michaelachrisco) [reply best-practices/database/README.md:4]: I totally missed that. I made the change before the announcement. 

Ill change it, thanks!

Comment (michaelachrisco) [reply best-practices/database/README.md]: fixed via: 4267c2a2ecbe5c136f87062bbc278fb541bafb89

File: best-practices/database/README.md (hunk 2 above)
Comment (michaelachrisco) [reply best-practices/database/README.md:6]: How does:
```suggestion
If you and your team is starting a new project or deciding on a database to use, contact architect@bitwiseindustries.com.
```
look. That way its more explicit as well as covering two common scenarios.

Comment (michaelachrisco) [reply best-practices/database/README.md]: Changed via: f277d483354669343f0b1c7c57e69313d9e782d7

Comment (Karvel) [reply best-practices/database/README.md]: If you and your team are starting a new project or deciding on a database to use, contact architect@bitwiseindustries.com.

Comment (michaelachrisco) [reply best-practices/database/README.md]: Grammar is important. 

Fixed via: a1397fe413e144cbcba9e709633b68536fbeb2e1

File: best-practices/database/README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
-# DATABASE
+# Databases
+A database is an organized collection of data, generally stored and accessed electronically from a computer system. Where databases are more complex they are often developed using formal design and modeling techniques. 
+
+At Bitwise Industries, we generally use T-SQL and NoSQL databases in our projects. More information can be found [here](https://www.altexsoft.com/blog/business/comparing-database-management-systems-mysql-postgresql-mssql-server-mongodb-elasticsearch-and-others/).
```
Comment (justincohan) [inline best-practices/database/README.md:4]: I didn't realized we had projects that used T-SQL

Comment (michaelachrisco) [reply best-practices/database/README.md:4]: I kinda used this as a blanket term....but now that you brought it up, I think this is incorrect.

T-SQL is just something I got used to saying at an old job, but it really should be SQL.

Comment (michaelachrisco) [reply best-practices/database/README.md]: fixed via: 94db7296ec241a09378c4746d3ce99efcf8e304a

**Review by justincohan — APPROVED** [review]

**Review by Karvel — APPROVED** [review]
Nice additions.

**Pull request merged**